components/user_settings/display/index.ts

```diff
@@ -15,23 +16,24 @@ import {getUserCurrentTimezone} from 'mattermost-redux/utils/timezone_utils';
 
 import {Preferences} from 'utils/constants';
 
-import UserSettingsDisplay from './user_settings_display.jsx';
+import {GlobalState} from 'types/store';
 
-function mapStateToProps(state) {
+import UserSettingsDisplay from './user_settings_display';
+
+function mapStateToProps(state: GlobalState) {
     const config = getConfig(state);
     const timezones = getTimezones(state);
     const currentUserId = getCurrentUserId(state);
     const userTimezone = getUserTimezone(state, currentUserId);
     const automaticTimezoneNotSet = userTimezone && userTimezone.useAutomaticTimezone && !userTimezone.automaticTimezone;
     const shouldAutoUpdateTimezone = !userTimezone || automaticTimezoneNotSet;
-
     const allowCustomThemes = config.AllowCustomThemes === 'true';
     const enableLinkPreviews = config.EnableLinkPreviews === 'true';
-    const defaultClientLocale = config.DefaultClientLocale;
+    const defaultClientLocale = config.DefaultClientLocale as string;
```

Comment:
Both this field and the one below it should already be of type `string`. How come we need the cast here ?

Reply:
there is only one line of defaultClineLocale, it is needed as I am getting an error while type checking.